docs/src/inverse_problems/structural_identifiability.md

```diff
@@ -0,0 +1,168 @@
+```example si1
+using Catalyst, Logging, StructuralIdentifiability
+goodwind_oscillator = @reaction_network begin
+    (pₘ/(1+P), dₘ), 0 <--> M
+    (pₑ*M,dₑ), 0 <--> E
+    (pₚ*E,dₚ), 0 <--> P
+end
+assess_identifiability(goodwind_oscillator; measured_quantities=[:M], loglevel=Logging.Error)
+```
+From the output, we find that `E(t)`, `pₑ`, and `pₚ` (the trajectory of $E$, and the production rates of $E$ and $P$, respectively) are non-identifiable. Next, `dₑ` and `dₚ` (the degradation rates of $E$ and $P$, respectively) are locally identifiable. Finally, `P(t)`, `M(t)`, `pₘ`, and `dₘ` (the trajectories of `P` and `M`, and the production and degradation rate of `M`, respectively) are all globally identifiable. We note that we also imported the Logging package, and provided the `loglevel=Logging.Error` input argument StructuralIdentifiability functions generally provides a large number of output logging messages. Hence, we will use this argument (which requires the Logging package) throughout this tutorial.
+
+
+Next, we can assess identifiability in the case where we can measure all three species concentrations:
+```example si1
+assess_identifiability(goodwind_oscillator; measured_quantities=[:M, :P, :E], loglevel=Logging.Error)
+```
+in which case all initial conditions and parameters become identifiable.
+
+
+### Indicating known parameters
+In the previous case we assumed that all parameters are unknown, however, this is not necessarily true. If there are parameters which value's are known, we can supply these using the `known_p` argument. Indeed, this might turn other, previously unidentifiable, parameters identifiable. Let us consider the previous example, where we measure the concentration of $M$ only, but also happen to know the production rate of $E$ ($pₑ$):
+```example si1
+assess_identifiability(gwo; measured_quantities=[:M], known_p=[:pₑ], loglevel=Logging.Error)
+```
+Not only does this turn the previously non-identifiable `pₑ` (globally) identifiable (which is obvious, given that its value is now known), but this additional information improve identifiability for several other network components.
+
+To, in a similar manner, indicate that certain initial conditions are known is a work in progress. Hopefully the feature should be an available in the near future.
+
+### Providing non-trivial measured quantities
+Sometimes, we are not actually measuring species species, but rather some combinations of species (or possibly parameters). Here, any algebraic expression can be used in `measured_quantities`. If so, used species and parameters have to first be `@unpack`'ed from the model. Say that we have a model where an enzyme ($E$) is converted between an active and inactive form, which in turns activates the production of a product, $P$:
```

Comment:
```suggestion
Sometimes, we are not actually measuring species, but rather combinations of species (or possibly parameters), which can be captured by `measured_quantities`, which allows any algebraic expression to be provided. Species and parameters have to first be `@unpack`'ed from the model to be used in such expressions. Say that we have a model where an enzyme ($E$) is converted between an active and inactive form, which in turns activates the production of a product, $P$:
```
Why do species and parameters have to be `@unpack`'ed? You can't do `sys.symbol`?

Reply:
Think I preferred the old start ratehr than "ometimes, we are not actually measuring species, but rather combinations of species (or possibly parameters), which can be captured by `measured_quantities`, which allows any algebraic expression to be provided". changing the other locally and pushing.

Reply:
Feel free to reword from the old version or my version, but please fix that the old version could be read as two completely independent, disconnected sentences as originally written.